### PR TITLE
Clean Finding Title for pwn_sast Parser

### PR DESCRIPTION
**Pull request by ninp0**

### CODE REVIEW

**Review by damiencarol — CHANGES_REQUESTED** [review]
Please add a unit test to show the change

**Comment by ninp0** [comment]
@damiencarol , can you expand on what you mean to add a unit test to show the change?  I'm wondering what should be added to the unit test that already exists in:
https://github.com/DefectDojo/django-DefectDojo/blob/dev/unittests/tools/test_pwn_sast_parser.py 

I'm not sure what should be added to test the change of the finding title.  Thanks in advance for your help.

**Comment by damiencarol** [comment]
take a look at the unit tests in `unittests` folder. Some are dedicated to this parser.
I can't help you directly has you restricted access to your branch :/

**Comment by ninp0** [comment]
Then would you be open to approving?  I'm the original author of this parser and I simply changed the string value of finding title.  What am I missing?

**Comment by damiencarol** [comment]
yes, I will approve once it's ready. the pb here is some parts are missing.
1. you are modifying title data, so you need a unit tests that check that.
2. you still have some Flake8 anomaly. you're using tabs but DefectDojo source code requires spaces.

File: unittests/tools/test_pwn_sast_parser.py [hunk 1]
```diff
@@ -31,3 +31,10 @@ def test_one_dup_finding(self):
         findings = parser.get_findings(testfile, Test())
         self.assertIsInstance(findings, list)
         self.assertEqual(1, len(findings))
+
+    def test_title_is_not_none(self):
+      testfile = open("unittests/scans/pwn_sast/one_finding.json")
+      parser = PWNSASTParser()
+      findings = parser.get_findings(testfile, Test())
+      self.assertIsInstance(findings, list)
+      assert findings.title is not None
```
Comment (damiencarol) [inline unittests/tools/test_pwn_sast_parser.py]: Test the value please
```suggestion
      self.assertEquals("...", findings.title)
```

**Review by damiencarol — CHANGES_REQUESTED** [review]

**Comment by damiencarol** [comment]
@ninp0 you really need to configure your tool to be aligned with DD code format.

**Comment by ninp0** [comment]
@damiencarol - change introduced...let me know if there's anything else needed.  Thanks.

**Review by damiencarol — APPROVED** [review]

**Review by Maffooch — APPROVED** [review]

**Pull request merged**